docs/connector-development/connector-builder-ui/record-processing.mdx

```diff
@@ -0,0 +1,339 @@
+import Diff from './assets/record-processing-schema-diff.png';
+
+# Record processing
+
+Builder connectors always make HTTP requests, receive the responses and emit records. Besides making the right requests, it's important to properly hand over the records to the system:
```

Comment:
Is "Builder connector" a term we use elsewhere? I don't hate the name, but we should standardize what we call these connectors so avoid confusion

Reply:
Not really, changed it to "Connectors built with the connector builder"